homeassistant/components/jewish_calendar/__init__.py

```diff
@@ -129,13 +131,16 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
         CONF_HAVDALAH_OFFSET_MINUTES, DEFAULT_HAVDALAH_OFFSET_MINUTES
     )
 
-    location = Location(
-        name=hass.config.location_name,
-        diaspora=diaspora,
-        latitude=config_entry.data.get(CONF_LATITUDE, hass.config.latitude),
-        longitude=config_entry.data.get(CONF_LONGITUDE, hass.config.longitude),
-        altitude=config_entry.data.get(CONF_ELEVATION, hass.config.elevation),
-        timezone=config_entry.data.get(CONF_TIME_ZONE, hass.config.time_zone),
+    location = await hass.async_add_executor_job(
+        partial(
+            Location,
+            name=hass.config.location_name,
+            diaspora=diaspora,
+            latitude=config_entry.data.get(CONF_LATITUDE, hass.config.latitude),
+            longitude=config_entry.data.get(CONF_LONGITUDE, hass.config.longitude),
+            altitude=config_entry.data.get(CONF_ELEVATION, hass.config.elevation),
+            timezone=config_entry.data.get(CONF_TIME_ZONE, hass.config.time_zone),
+        )
     )
 
     hass.data.setdefault(DOMAIN, {})[config_entry.entry_id] = {
```

Comment:
This integration could be simplified to use `entry.runtime_data` in a future PR

https://developers.home-assistant.io/blog/2024/04/30/store-runtime-data-inside-config-entry